components/reusable-components/footer/footer.module.scss

```diff
@@ -0,0 +1,136 @@
+@mixin respond-to($size) {
+  @if $size == 'small' {
+    @media (max-width: 768px) {
+      @content;
+    }
+  } @else if $size == 'medium' {
+    @media (min-width: 769px) and (max-width: 1024px) {
+      @content;
+    }
+  } @else if $size == 'large' {
+    @media (min-width: 1025px) and (max-width: 1440px) {
+      @content;
+    }
+  } @else if $size == 'extra-large' {
+    @media (min-width: 1441px) {
+      @content;
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  background-color: var(--bg-color);
+  color: #fff;
+}
+
+.footerBrand {
+  flex: 1;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  padding-left: 1.5em;
+  padding-bottom: 6.5em;
+}
+
+.contactSection p:first-child {
+  flex: 2;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 700;
+  margin-right: 5em;
+  margin-left: 16em;
+  margin-bottom: 1.3em;
+}
+
+.contactSection p:last-child {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  margin-right: 6.25em;
+  padding-left: 16em;
+  padding-bottom: 8em;
+  text-decoration: underline;
+}
+
+.socialMediaSection {
+  flex: 2;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 700;
+  text-transform: uppercase;
+  padding-bottom: 2.5em;
+  padding-left: 4em;
+}
+
+.socialMediaSection p {
+  padding-bottom: 1.3em;
+}
+
+.socialMediaSection a {
+  display: flex; // Use flex to allow centering the icon horizontally
+  align-items: center; // Center the icon vertically
+  margin-bottom: 12px;
+  color: #fff;
+  text-decoration: none;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  gap: 10px; // Add space between the icon and text
+}
+
+.newsletterSection {
+  flex: 2;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 700;
+  padding-bottom: 8em;
+}
+
+.newsletterSection p {
+  padding-bottom: 1.3em;
+}
+
+.newsletterSection input {
+  background-color: var(--bg-color);
+  border: 1px solid var(--text-color);
+  padding: 0.5em;
+  cursor: text;
+  width: 60%;
+  max-width: 300px;
+}
+
+.newsletterSection input::placeholder {
+  color: var(--text-color);
```

Comment:
@Mikivitkovski did you test the color for the text (e-mail) we put in the input?